__tests__/App-test.js

```diff
@@ -1,14 +1,215 @@
-/**
- * @format
- */
-
-import 'react-native';
 import React from 'react';
-import App from '../App';
+import {create, act} from 'react-test-renderer';
+import {ButtonExamplePage} from '../src/examples/ButtonExamplePage';
+import {TextExamplePage} from '../src/examples/TextExamplePage';
+import {TextInputExamplePage} from '../src/examples/TextInputExamplePage';
+import {TouchableHighlightExamplePage} from '../src/examples/TouchableHighlightExamplePage';
+import {TouchableOpacityExamplePage} from '../src/examples/TouchableOpacityExamplePage';
+import {SwitchExamplePage} from '../src/examples/SwitchExamplePage';
+import {ViewExamplePage} from '../src/examples/ViewExamplePage';
+import {ImageExamplePage} from '../src/examples/ImageExamplePage';
+import {PressableExamplePage} from '../src/examples/PressableExamplePage';
+import {FlatListExamplePage} from '../src/examples/FlatListExamplePage';
+import {ScrollViewExamplePage} from '../src/examples/ScrollViewExample';
+import {PopupExamplePage} from '../src/examples/PopupExamplePage';
+import {FlyoutExamplePage} from '../src/examples/FlyoutExamplePage';
+import {CheckBoxExamplePage} from '../src/examples/CheckBoxExamplePage';
+import {ConfigExamplePage} from '../src/examples/ConfigExamplePage';
+import {DatePickerExamplePage} from '../src/examples/DatePickerExamplePage';
+import {TimePickerExamplePage} from '../src/examples/TimePickerExamplePage';
+//import {SketchExamplePage} from '../src/examples/SketchExamplePage';
```

Comment:
for the sake of leaving some breadcrumbs, do you know what the issue was with including this and track player?

Reply:
Yes I can add comments!